OSDOCS-16939: DITA abstracts — OLM understanding topics (#115281)
openshift/openshift-docs: modules/olm-operatorgroups-csv-annotations.adoc

[id="olm-operatorgroups-csv-annotations_{context}"]
= Operator group CSV annotations

[role="_abstract"]

> Member CSVs of an Operator group carry annotations that identify the group name, namespace, and target namespace selection. OLM applies these annotations to member CSVs and omits the target namespace annotation on copied CSVs to prevent duplication across tenants.

Reviewer: ```suggestion
Member cluster service versions (CSVs) of an Operator group carry annotations that identify the group name, namespace, and target namespace selection. 
```
